cpp/trie.h

```diff
@@ -25,6 +25,8 @@ class Trie {
 
   bool IsWord() const { return is_word_; }
   void SetIsWord() { is_word_ = true; }
+  // Bitmap of children, 1 << 0..25
+  uint32_t HasChildren() const { return has_children_; }
```

Comment:
Suggest calling this `ChildMask`. `HasChildren` sounds like it should return a boolean indicating whether this node has children.